modules/statics/src/coins.ts

```diff
@@ -262,6 +263,7 @@ const COSMOS_SIDECHAIN_FEATURES = [
   CoinFeature.INCREASED_TX_REQUEST_REBUILD_LIMIT,
   CoinFeature.BULK_TRANSACTION,
   CoinFeature.BULK_STAKING_TRANSACTION,
+  CoinFeature.STUCK_TRANSACTION_MANAGEMENT,
```

Comment:
Are we going to add this in solana as well?

Reply:
no, adding it for chains that support DKLS